src/test/osdc/object_cacher_stress.cc

```diff
@@ -351,6 +351,110 @@ int correctness_test(uint64_t delay_ns)
   return EXIT_FAILURE;
 }
 
+int flush_test()
```

Comment:
This is not the place to put this. Let's make a new `src/test/osdc/object_cacher_misc.cc` and put it there for now. You'll need to add to the CMakeLists.txt, etc.

Reply:
Also make `qa/workunits/osdc/object_cacher_misc.sh` and add it to:

qa/suites/rados/objectstore/backends/objectcacher-stress.yaml

(at the end) so it gets run in QA.